StitchCore/StitchCore/Source/Auth/Providers/Anonymous/AnonymousAuthProvider.swift

```diff
@@ -3,10 +3,10 @@ import ExtendedJson
 /// AnonymousAuthProvider provides a way to authenticate anonymously.
 public struct AnonymousAuthProvider: AuthProvider {
     /// The authentication type for anonymous login.
-    public let type: String = "anon"
+    public let type: String = "anon-user"
 
     /// The name for anonymous login
-    public let name: String = "user"
+    public let name: String = "anon-user"
```

Comment:
[minor] The name is technically configurable. The name if the response from auth providers is the name you use in the URL, not the type.